docs/design/StratisSoftwareDesign.lyx

```diff
@@ -4666,16 +4692,96 @@ OS Integration: /dev entries
 
 \begin_layout Standard
 Stratis allows the user to create filesystems, which then can be mounted
- and used via mount(8) and the fstab(5).
- Stratisd provides a udev rule that generates a /dev/stratis directory.
- It creates /dev/stratis/<poolname> for each pool present on the system,
- and /dev/stratis/<poolname>/<filesystemname> for each filesystem within
- the pool.
+ and used via 
+\begin_inset Flex Code
+status open
+
+\begin_layout Plain Layout
+mount(8)
+\end_layout
+
+\end_inset
+
+ and the 
+\begin_inset Flex Code
+status open
+
+\begin_layout Plain Layout
+fstab(5)
+\end_layout
+
+\end_inset
+
+.
+ Stratisd provides a udev rule that generates a 
+\begin_inset Flex Code
+status open
+
+\begin_layout Plain Layout
+/dev/stratis
+\end_layout
+
+\end_inset
+
+ directory.
+ It creates 
+\begin_inset Flex Code
+status open
+
+\begin_layout Plain Layout
+/dev/stratis/<poolname>
+\end_layout
+
+\end_inset
+
+ for each pool present on the system, and 
+\begin_inset Flex Code
+status open
+
+\begin_layout Plain Layout
+/dev/stratis/<poolname>/<filesystemname> 
```

Comment:
This still formats too annoyingly. Try inserting a justified line break right before this chunk of text.